### PR TITLE
Adds integration tests for the Async Firstore module

### PR DESCRIPTION
**Pull request by jkyle109**
Adds integration tests for the async Firstore module.

Related to https://github.com/firebase/firebase-admin-python/issues/104

### CODE REVIEW

File: integration/conftest.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Google Inc.
+# Copyright 2022 Google Inc.
```
Comment (lahirumaramba) [inline integration/conftest.py:1]: We only set this for new files (this is the creation date)

**Review by lahirumaramba — APPROVED** [review]
LGTM!

**Comment by lahirumaramba** [comment]
Added `staging` label to trigger integration tests

File: integration/conftest.py (hunk 1 above)
Comment (jkyle109) [reply integration/conftest.py]: Got it, thought this was last time edited. I'll keep that in mind!

**Pull request merged**